pxteditor/events.ts

```diff
@@ -0,0 +1,20 @@
+namespace pxt.events {
```

Comment:
can you put this under ``pxt.editor.events`` ? I think all editor stuff is there.

Reply:
sure.